### ISSUE TITLE
Mixtral Model Download getting stuck

### ISSUE THREAD

**Issue by RonanKMcGovern** [huggingface/text-generation-inference#1413]
### System Info

Using the latest docker image

### Information

- [X] Docker
- [ ] The CLI directly

### Tasks

- [X] An officially supported command
- [ ] My own modifications

### Reproduction

Run [this Runpod Template](https://runpod.io/gsc?template=1ydpo4766w&ref=jmfkcdio
).

The model gets stuck at the 6th shard. This requires a pod re-start and multiple later re-starts to get all shards downloaded.

### Expected behavior

I would expect all weights to download smoothly.

What's possible is that the download isn't stopping entirely but progressing at a very very slow pace. I have seen this to some degree when working in Jupyter notebooks, but these are 5 GB shards and the rate of progress does seem to stall with TGI.

Runpod Template Details:
```
ghcr.io/huggingface/text-generation-inference:latest

--model-id mistralai/Mixtral-8x7B-Instruct-v0.1 --trust-remote-code --port 8080 --max-input-length 2048 --max-total-tokens 4096 --max-batch-prefill-tokens 4096 --quantize eetq
```

As a related matter, why does TGI not appear to download the shards in parallel?

**Comment by OlivierDehaene** [comment]
Duplicate of #1186
